@xen-orchestra/lite/src/components/Table.tsx

```diff
@@ -19,55 +52,318 @@ interface ParentState {}
 
 interface State {}
 
-interface Props {
-  collection: Item[] | undefined
-  columns: Column<any>[]
-  placeholder?: JSX.Element
+interface StatePagination {}
+
+type Props = IsSelectable &
+  RouteComponentProps & {
+    collection: Item[] | undefined
+    columns: Column<any>[]
+    dataType?: string
+    stateUrlParam: string
+  }
+
+interface PropsPagination {
+  dataType?: string
+  nItemsOnThePage: number
+  nSelectedItems?: number
+  nTotalItems: number
+  nTotalPage: number
+  onPaginationChange: (_: any, page: number) => void
+  onShowByChange: (value: SelectChangeEvent<unknown>) => void
+  page: number
+  reverse?: boolean
+  showByValue: number
```

Comment:
why?

Reply:
This prop is required for the calculation of `x-y of z VMs`